documentation/SOFTWARE_REQUIREMENTS_SPEC.md

```diff
@@ -14,12 +14,12 @@ Our web-based Sokoban game will have greater user reachability compared to a mor
 
 ### Phase 1: Basic Gameplay Requirements
 
-#### World
+#### Game Board
 
-* The world shall be approximately 500 pixels tall by 600 pixels wide.
-* The world shall have top view perspective (viewed from above). 
-* Every element in the world shall have a graphical dimensions of 20 pixels by 20 pixels.
-* The world shall contain the following:
+* The board shall be approximately 500 pixels tall by 600 pixels wide.
```

Comment:
Will this work for mobile (or do we even care about mobile -- I vote yes)? Do we need to specify the dimensions? If so, how did we arrive at the 600x500 choice?

Reply:
The dimension is arbitrary. What would be a better default dimension?

Reply:
I suggest we avoid committing to some arbitrary values we have no justification for. I think it's better to skip it instead of specify unless we're >90% sure of the value, or we have some sort of rationale to back it up, or we can more clearly state that they're arbitrary.